### ISSUE TITLE
Running in interactive mode for testing

### ISSUE THREAD

**Issue by mappuji** [exercism/common-lisp#180]
I am following this http://exercism.io/languages/common-lisp/tests

When I try `clisp hamming-test.lisp`

I got 
```
*** - READ from #<INPUT BUFFERED FILE-STREAM CHARACTER #P"hamming-test.lisp" @1>: there is no package with name "QL"
```

I think better to give the newcomers direction to run the test with `clisp -i hamming-test.lisp`

**Comment by serialhex** [comment]
Do you have quicklisp installed?  If not, follow this http://exercism.io/languages/common-lisp/installation and install quicklisp, as that should solve your problem.

It shouldn't have to be loaded with `-i` in order to work. Does it work with one and not the other?

**Comment by mappuji** [comment]
I am sure that I have `clisp` installed and working in my computer, but I can't have the test running without interactive mode. I am using Windows 10. Maybe it is operating system-specific issue. Will try on my mac.

**Comment by verdammelt** [comment]
@mappuji The error appears to be because QuickLisp, a program which helps with packaging in Lisp is not installed in your CLisp. This would not have been done when you installed CLisp.

Instructions on how to install QuickLisp can be found in our track's installation instructions (http://exercism.io/languages/common-lisp/installation) and on QuickLisp's own site (https://www.quicklisp.org/beta/#installation).

If the instructions are unclear or do not work let me know so we can improve them.

**Comment by mappuji** [comment]
@verdammelt I also sure that I have installed the QuickLisp, is there is any specific way to install CommonLisp to install that?

**Comment by verdammelt** [comment]
If you followed the instructions on the track's page then that should include QuickLisp.

Have you had a chance to try this on a Mac like you suggested above? It would be interesting to see if this is a Windows vs. Unix/Mac problem.

**Comment by verdammelt** [comment]
So I could replicate the problem on a Mac. This is not a problem of Windows.

Clisp does not read in the user's init file when you provide a file on the command line (i.e. `clisp foo.lisp`) thus quicklisp is not loaded (as that is done in the init file).

There are two options:

* `clisp -i ~/.clisprc.lisp hamming-test.lisp` 
* `clisp -i hamming-test.lisp -x '(ext:exit)'`

Both will cause clisp to load the config file, run the tests and exit.

Which do you think would be preferable to use as the suggested method? Is there another option?

**Comment by mappuji** [comment]
Oh thats cool, thank you @verdammelt

**Comment by verdammelt** [comment]
@mappuji Can you create a PR with your suggested doc change? Or at least let me know where these instructions for Clisp might do the most good? I don't see any instruction about running the test files from the command line for any implementation at the moment...

**Comment by mappuji** [comment]
@verdammelt will do today

**Comment by verdammelt** [comment]
Fixed with #185
